backend/.env.example

```
@@ -20,6 +20,7 @@ REDIS_URL=redis://localhost:6379/0
 # Security
 FERNET_KEY=
 CORS_ORIGINS=http://localhost:3000
+CANARY_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CANARY_TOKEN key should go before the CORS_ORIGINS key

(UnorderedKey)